pkg/kubelet/apis/config/v1beta1/doc.go

```diff
@@ -16,7 +16,9 @@ limitations under the License.
 
 // +k8s:deepcopy-gen=package
 // +k8s:conversion-gen=k8s.io/kubernetes/pkg/kubelet/apis/config
-// +k8s:openapi-gen=true
+// +k8s:conversion-gen-external-types=k8s.io/kubelet/config/v1beta1
```

Comment:
is this necessary to point to itself?

Reply:
it's not pointing to itself, it's pointing to the staging types from the internal k8s.io meta pkg

Reply:
oh, not itself. nvmd.